### PR TITLE
Code Reviewed

### PR DESCRIPTION
**Pull request by Nikhil8869**
1. I have implemented the required changes in the code to introduce a responsive navigation bar in the previous iteration, where initially there was no responsive navigation bar.
2. I have updated the code to ensure that the header and navigation bar remain fixed at the top of the page, even when the website is scrolled.
3. I have applied border-radius changes to the images within the 'Safest Neighborhood' section, ensuring a rounded appearance. Additionally, I have centered the text beneath each element within the section

### CODE REVIEW

**Comment by Nikhil8869** [comment]
1. **CHANGES NEEDED:**Header position is not fixed.
filename: _Layout.cshtml
code: `<header class="header_section fixed-top bg-dark">`
cititations: https://getbootstrap.com/docs/5.0/helpers/position/

2.**CHANGES NEEDED:**Made Navigation Bar responsive and visible in mobile view
filename: _Layout.cshtml
code: `<nav class="navbar navbar-expand-lg custom_nav-container navbar-expand-sm">`
cititations: https://getbootstrap.com/docs/5.0/components/navbar/#responsive-behaviors

3.**CHANGES NEEDED:**Made borders of all the images to center and added some of the margin on every text element
filename: index.cshtml
code: `image of place
        <h4 class="h4 text-center mt-2">Sayler Park</h4>`
cititations: https://getbootstrap.com/docs/4.0/utilities/borders/

File: Pages/Index.cshtml [hunk 1]
```diff
@@ -56,7 +56,7 @@
 
 @if (Model.SearchResults != null)
 {
-    <div class="search-results">
+    <div class="search-results border-dark">
```
Comment (discospiff) [inline Pages/Index.cshtml:59]: Does this reduce technical debt?  If so, how so?  Is there an accessibility issue here that this fixes?
Code reviews should focus on alignment with best practices.  Unless there is a tech debt related reason to do so, UI changes should be left up to design.

File: Pages/Index.cshtml [hunk 2]
```diff
@@ -69,35 +69,35 @@
 
 
         </ul>
-        <hr> <!--placed horizontal line outside-->
+         <!--placed horizontal line outside-->
```
Comment (discospiff) [inline Pages/Index.cshtml:72]: Why was the hr removed, and the comment remains?  This comment is now misleading.

File: Pages/Index.cshtml [hunk 3]
```diff
@@ -69,35 +69,35 @@
...
 }
 
-<h3 class="text-center w-100 text-dark h3 mt-5 mb-2">Safest Neighborhoods In Cincinnati</h3>
+<h3 class="text-center w-100 text-dark h3 mt-5 mb-2 ">Safest Neighborhoods In Cincinnati</h3>
 <div class="row g-3">
     <div class="col-4 ">
-        <img src="https://i0.wp.com/haushomemagazine.com/wp-content/uploads/2021/04/My-Post-6.jpg?w=1200&ssl=1" class="zoom-hover rounded-top" alt="image of place" />
-        <h4 class="h4">Hyde Park, Cincinnati</h4>
+        <img src="https://i0.wp.com/haushomemagazine.com/wp-content/uploads/2021/04/My-Post-6.jpg?w=1200&ssl=1" class="zoom-hover rounded" alt="image of place" />
+        <h4 class="h4 text-center">Hyde Park, Cincinnati</h4>
```
Comment (discospiff) [inline Pages/Index.cshtml:80]: Use linked stylesheets instead of inline.

File: Pages/Shared/_Layout.cshtml [hunk 4]
```diff
@@ -41,24 +41,30 @@
...
+                        <ul class="navbar-nav">
                             <li class="nav-item active">
                                 <a class="nav-link" asp-page="Index">Home <span class="sr-only">(current)</span></a>
                             </li>
+                            <li class="nav-item active">
+                                <a class="nav-link" asp-page="About"> About <span class="sr-only">(current)</span></a>
+                            </li>
+                            <li class="nav-item active">
+                                <a class="nav-link" asp-page="Contact"> Contact <span class="sr-only">(current)</span></a>
+                            </li>
```
Comment (discospiff) [inline Pages/Shared/_Layout.cshtml:67]: -Are there pages for these links?  Avoid adding unimplemented features, as unused code is clutter, which is technical debt.
- Do you really need these pages?  Avoid adding additional pages, which aren't adding much value, and will distract the user.

**Pull request merged**